javascript-arrays/main.js

```diff
@@ -0,0 +1,24 @@
+const colors = ['red', 'white', 'blue'];
+console.log('Value of colors[0] ', colors[0]);
+console.log('Value of colors[1] ', colors[1]);
+console.log('Value of colors[2] ', colors[2]);
+
+const America = 'America is ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.';
+console.log(America);
+
+colors[2] = 'green';
+const Mexico = 'Mexico is ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.';
+console.log(Mexico);
+
+console.log('Value of colors ', colors);
+
+const students = ['Chase', 'Kaine', 'Spencer', 'Cameron'];
+const numberOfStudents = students.length;
+
+console.log('There are ', numberOfStudents, ' students in the class.');
+
+const lastIndex = numberOfStudents - 1;
+const lastStudent = students[lastIndex];
+console.log('The last student in class is ', lastStudent);
```

Comment:
Compare this sentence to the sentence in our example output in the instructions!

Reply:
![image](https://user-images.githubusercontent.com/126129416/226689425-393e12d9-1ecc-4810-9a7f-59102994edc2.png)